terraform/paas/terraform_qa.tfvars

```diff
@@ -0,0 +1,15 @@
+app = {
+  name         = "qa-publish-postgraduate-teacher-training"
+  docker_image = "dfedigital/publish-teacher-training:latest"
+  hostname     = "qa-publish-postgraduate-teacher-training"
+  space        = "find-qa"
+}
+
+app_env = {
```

Comment:
There are more environment variables in the azure template: https://github.com/DFE-Digital/publish-teacher-training/blob/master/azure/template.json#L296
DOn't we need them here?

Reply:
Added